src/server/api/controllers/modules.controller.crop-stages-endpoint.js

```diff
@@ -0,0 +1,24 @@
+const knex = require('../../config/db');
+
+const getCropStages = async () => {
+  try {
+    const startDate = await knex('batches')
+      .select('seeding_date')
+      .where({ id: '31' });
```

Comment:
This should not be hardcoded, it should be passed as a parameter :)

Reply:
yes it was just a test to see the behavior getting data because we are not clear . on what bases we need to select .. is it most recent batch ? what about if we have more then one batch .??